bin/node-template/README.md

```diff
@@ -1,97 +1,71 @@
-[Wasm](https://substrate.dev/docs/en/knowledgebase/advanced/executor#wasm-execution) and
-[native](https://substrate.dev/docs/en/knowledgebase/advanced/executor#native-execution) code:
+The `cargo run` command will perform an initial build. Use the following command to build the node
+without launching it:
 
-```bash
+```sh
 cargo build --release
 ```
 
-## Run
-
-### Single Node Development Chain
+### Embedded Docs
 
-Purge any existing dev chain state:
+Once the project has been built, the following command can be used to explore all parameters and
+subcommands:
 
-```bash
-./target/release/node-template purge-chain --dev
+```sh
+./target/release/node-template -h
 ```
 
-Start a dev chain:
+## Run
 
-```bash
-./target/release/node-template --dev
-```
+The provided `cargo run` command will launch a temporary node and its state will be discarded after
+you terminate the process. After the project has been built, there are other ways to launch the
+node.
 
-Or, start a dev chain with detailed logging:
+### Single-Node Development Chain
+
+This command will start the single-node development chain with persistent state:
 
 ```bash
-RUST_LOG=debug RUST_BACKTRACE=1 ./target/release/node-template -lruntime=debug --dev
+./target/release/node-template --dev
 ```
 
-### Multi-Node Local Testnet
-
-To see the multi-node consensus algorithm in action, run a local testnet with two validator nodes,
-Alice and Bob, that have been [configured](./node/src/chain_spec.rs) as the initial
-authorities of the `local` testnet chain and endowed with testnet units.
-
-Note: this will require two terminal sessions (one for each node).
-
-Start Alice's node first. The command below uses the default TCP port (30333) and specifies
-`/tmp/alice` as the chain database location. Alice's node ID will be
-`12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp` (legacy representation:
-`QmRpheLN4JWdAnY7HGJfWFNbfkQCb6tFf4vvA6hgjMZKrR`); this is determined by the `node-key`.
+Purge the development chain's state:
 
 ```bash
-cargo run -- \
-  --base-path /tmp/alice \
-  --chain=local \
-  --alice \
-  --node-key 0000000000000000000000000000000000000000000000000000000000000001 \
-  --telemetry-url 'ws://telemetry.polkadot.io:1024 0' \
-  --validator
+./target/release/node-template purge-chain --dev
 ```
 
-In another terminal, use the following command to start Bob's node on a different TCP port (30334)
-and with a chain database location of `/tmp/bob`. The `--bootnodes` option will connect his node to
-Alice's on TCP port 30333:
+Start the development chain with detailed logging:
 
 ```bash
-cargo run -- \
-  --base-path /tmp/bob \
-  --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp \
-  --chain=local \
-  --bob \
-  --port 30334 \
-  --ws-port 9945 \
-  --telemetry-url 'ws://telemetry.polkadot.io:1024 0' \
-  --validator
+RUST_LOG=debug RUST_BACKTRACE=1 ./target/release/node-template -lruntime=debug --dev
```

Comment:
FYI we also support `-ldebug`. Basically anything that you put in front of RUST_LOG can go with -l